textdb/textdb-common/src/main/java/edu/uci/ics/textdb/common/utils/Utils.java

```diff
@@ -32,78 +32,86 @@
 import edu.uci.ics.textdb.common.field.TextField;
 
 public class Utils {
-    public static IField getField(FieldType fieldType, String fieldValue) throws ParseException{
+    public static IField getField(FieldType fieldType, String fieldValue) throws ParseException {

```

Comment:
I think the formatting changes github shows is because of the individual formatting settings in eclipse. So now if I make a change to a file in eclipse, eclipse automatically formats and indents the code according to the settings. Actually, there are no formatting changes, all the changes we made are logical changes. I'll check if there's a way to get around this issue.